db-commons/src/main/resources/reference.conf

```diff
@@ -1,4 +1,9 @@
 bitcoin-s {
     datadir = ${HOME}/.bitcoin-s
     network = regtest # regtest, testnet3, mainnet
+
+    # settings for wallet module
+    wallet {
+        defaultAccountType = legacy # legacy, segwit, nested-segwit
```

Comment:
not necessarily related to this PR, but we should really change the segwit key to `segwitv0` or something like that

Reply:
Not sure I agree on that point, at least for the time being. I'm a big believer in the ["principle of least surprise"](https://en.wikipedia.org/wiki/Principle_of_least_astonishment). In that spirit I think following convention for naming things makes it a lot easier for users. We could optionally make both `segwit` and `segwitv0` map to the same thing, etc

Reply:
Once a taproot wallet proposal comes out, we could call that `taproot` (or whatever the canonical name becomes) + `segwitv1`